unit-tests/src/test/scala/scala/scalanative/regex/PatternSuite.scala

```diff
@@ -403,13 +401,11 @@ object PatternSuite extends tests.Suite {
   }
 
   private def syntax(pattern: String, description: String, index: Int): Unit = {
-    assertThrowsAnd[PatternSyntaxException](Pattern.compile(pattern))(
-      e => {
-        e.getDescription == description &&
-        e.getPattern == pattern
-        e.getIndex == index
-      }
-    )
+    assertThrowsAnd[PatternSyntaxException](Pattern.compile(pattern))(e => {
+      e.getDescription == description &&
+      e.getPattern == pattern
```

Comment:
Missing `&&` here too.

Reply:
I suppose that could be missing but it wasn't in the original either. @LeeTibbert Perhaps this is some code you are familiar with?

Reply:
Thanks @LeeTibbert. See https://github.com/scala-native/scala-native/issues/1740